doc/classes/ProjectSettings.xml

```diff
@@ -652,6 +652,9 @@
 			See [enum DisplayServer.VSyncMode] for possible values and how they affect the behavior of your application.
 			Depending on the platform and used renderer, the engine will fall back to [code]Enabled[/code], if the desired mode is not supported.
 		</member>
+		<member name="editor/movie_writer/create_new_file" type="bool" setter="" getter="" default="true">
```

Comment:
I'd rename the setting to `editor/movie_writer/create_new_file_if_existing`, as MovieWriter will always create a new file by design :slightly_smiling_face: